src/test/java/com/internship/socialnetwork/service/PostServiceImplTest.java

```diff
@@ -0,0 +1,207 @@
+package com.internship.socialnetwork.service;
+
+import com.internship.socialnetwork.dto.PostDto;
+import com.internship.socialnetwork.exception.NotFoundException;
+import com.internship.socialnetwork.model.Post;
+import com.internship.socialnetwork.model.User;
+import com.internship.socialnetwork.repository.PostRepository;
+import com.internship.socialnetwork.service.impl.PostServiceImpl;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Optional;
+
+@ExtendWith(MockitoExtension.class)
+public class PostServiceImplTest {
+
+    private final Long id1 = 1L;
+
+    private final Long id2 = 2L;
+
+    private final Long id3 = 3L;
+
+    private final String text1 = "Text1";
+
+    private final String text2 = "Text2";
```

Comment:
Do we really need to create variables named like this for numbers and random text